tests/chainer_tests/functions_tests/math_tests/test_sign.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+
+
+class UnaryFunctionsTestBase(unittest.TestCase):
+
+    @property
+    def no_grads(self):
+        return (False,)
+
+    def make_data(self):
+        raise NotImplementedError
+
+    def setUp(self):
+        self.eps = 1e-3
+        while True:
```

Comment:
Is this necessary?

Reply:
No sorry, this is not necessary for the sign function. Removed!